package.json

```diff
@@ -12,7 +12,8 @@
     "precommit": "npm start precommit",
     "prepush": "npm start prepush",
     "postversion": "npm start postversion",
-    "prepublish": "npm start prepublish"
+    "prepublish": "npm start prepublish",
+    "e2e": "REPORT_DIR=${CIRCLE_TEST_REPORTS} LOG_DIR=${CIRCLE_ARTIFACTS} NODE_ENV=test nightwatch-autorun"
```

Comment:
this is what `ci.e2e` does. You may need to update `react-compoennt-template` version to the latest one.